src/Node_verify.cpp

```diff
@@ -397,12 +397,17 @@ void Node::verify_vdf_task(std::shared_ptr<const ProofOfTime> proof) const noexc
 {
 	std::lock_guard lock(vdf_mutex);
 
+	// TODO: remove temporary create/set 'verify_vdf_cpuopencl' variable when generated VNX ready
+	bool verify_vdf_cpuopencl = true;
+
 	const auto time_begin = vnx::get_wall_time_micros();
 	try {
 		for(int i = 0; i < 3; ++i) {
 			if(i < 2 || (verify_vdf_rewards && proof->reward_addr)) {
 				if(auto engine = opencl_vdf[i]) {
-					engine->compute(proof, i);
+					if(!(verify_vdf_cpuopencl && i < 1)) {
```

Comment:
`if(i > 0 || !verify_vdf_cpuopencl)` seems more readable to me ;)

Inverse logic is hard to get your head around...

Reply:
I'll buy that :) Done.